build.rs

```diff
@@ -125,85 +125,96 @@ fn build_c_code(out_dir: &str) -> Result<(), std::env::VarError> {
         });
         run_command_with_args(&make, &args);
     } else {
-        let arch = target_triple[0];
-        let (platform, optional_amd64) = match arch {
-            "i686" => ("Win32", None),
-            "x86_64" => ("x64", Some("amd64")),
-            _ => panic!("unexpected ARCH: {}", arch),
-        };
+        try!(build_msvc(&target_triple,
+                        disable_opt,
+                        &num_jobs,
+                        lib_path.clone(),
+                        out_dir));
```

Comment:
I don't know how to configure `rustfmt` to format this in the compact style, but it should be like this:

```rust
        build_msvc(&target_triple, disable_opt, &num_jobs, lib_path.clone(),
                   out_dir);
```